src/neon/mod.rs

```diff
@@ -0,0 +1,6 @@
+pub mod deser;
+pub mod stage1;
+pub mod utf8check;
+mod simd;
```

Comment:
adding a note here so we don't forget to remove that prior to merge / once the intrinsics are in nightly